### PR TITLE
Visit a business through the URL `/my-business/name`

### PR DESCRIPTION
**Pull request by Euraldius**
This PR MUST BE MERGED FIRST: https://github.com/thoughtbot/tinysplash/pull/11

### CODE REVIEW

**Comment by Euraldius** [comment]
The PR in the description has been merged, so this PR is good to go after it has been reviewed

File: app/views/application/_header.html.slim [hunk 1]
```diff
@@ -6,6 +6,6 @@ nav class="navigation--main"
         method: :delete,
         class: "button"
   - else
-    = link_to t("links.sign_in"),
+    == link_to t("links.sign_in"),
```
Comment (jasonramirez) [inline app/views/application/_header.html.slim]: I don't think == is necessary. I think a single = works just fine. (famous last words)

Comment (jasonramirez) [inline app/views/application/_header.html.slim]: Actually, a rebase should take care of this

File: config/routes.rb [hunk 2]
```diff
@@ -4,4 +4,6 @@
   devise_for :users
 
   resources :businesses
+
+  get "/:id" => "businesses#show"
```
Comment (jasonramirez) [inline config/routes.rb]: cool!

File: spec/features/business_url_spec.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+feature "User can visit a business's home page" do
+  scenario "using a friendly URL slug" do
+    create(:business, name: "Example Ltd")
+
+    visit "/example-ltd"
+
+    expect(page).to have_content "Example Ltd"
```
Comment (jasonramirez) [inline spec/features/business_url_spec.rb]: Love it! Not that this has to be part of the PR, but what about adding some dev seeds?

**Comment by jasonramirez** [comment]
Rebase will clear a couple of my comments, then hound and a final note about dev seeds. The code looks great! :hamburger:

File: spec/features/business_url_spec.rb (hunk 3 above)
Comment (Euraldius) [inline spec/features/business_url_spec.rb]: I believe these have now been added in a separate PR: https://github.com/thoughtbot/tinysplash/pull/15

**Comment by Euraldius** [comment]
@jasonramirez I think this is ready to merge, but because it is a little old now, I'd like one last look through!

**Pull request merged**